tests/contrib/django/django_tests.py

```diff
@@ -527,6 +529,77 @@ def test_request_metrics(self):
             self.assertEqual(timing['result'],
                              200)
 
+    def test_request_metrics_301_append_slash(self):
+        self.opbeat._requests_store.get_all()  # clear the store
+
+        # enable middleware wrapping
+        client = get_client()
+        client.wrap_django_middleware = True
+
+        with self.settings(
+            MIDDLEWARE_CLASSES=[
+                'opbeat.contrib.django.middleware.OpbeatAPMMiddleware',
+                'django.middleware.common.CommonMiddleware',
+            ],
+            APPEND_SLASH=True,
+        ):
+            self.client.get(reverse('opbeat-no-error-slash')[:-1])
+        timed_requests = self.opbeat._requests_store.get_all()
+        timing = timed_requests[0]
+        self.assertEqual(
+            timing['transaction'],
+            'django.middleware.common.CommonMiddleware.process_request'
+        )
+
+    def test_request_metrics_301_prepend_www(self):
+        self.opbeat._requests_store.get_all()  # clear the store
+
+        # enable middleware wrapping
+        client = get_client()
+        client.wrap_django_middleware = True
+
+        with self.settings(
+            MIDDLEWARE_CLASSES=[
+                'opbeat.contrib.django.middleware.OpbeatAPMMiddleware',
+                'django.middleware.common.CommonMiddleware',
+            ],
+            PREPEND_WWW=True,
+        ):
+            self.client.get(reverse('opbeat-no-error'))
+        timed_requests = self.opbeat._requests_store.get_all()
+        timing = timed_requests[0]
+        self.assertEqual(
+            timing['transaction'],
+            'django.middleware.common.CommonMiddleware.process_request'
```

Comment:
Any way we can give even more info? It seems like it would be useful to know when it was a PREPEND_WWW and when it was APPEND_SLASH?